### ISSUE TITLE
False positive for indent namespace error.

### ISSUE THREAD

**Issue by trivialfis** [cpplint/cpplint#293]
Hi, we are running into linter errors related to indent after upgrading cpplint to 2.0. I think the error is a false positive as it raises errors for parameter indentation.

test.cc:
``` c++
#include <iostream>
namespace mynamespace {
struct SuperLongTypeName {};

void fn(SuperLongTypeName a, SuperLongTypeName b, SuperLongTypeName c,
        SuperLongTypeName d) {
  std::cout << "foo" << std::endl;
}
}  // namespace mynamespace
```

error
```
./test.cc:6:  Do not indent within a namespace.  [whitespace/indent_namespace] [4]
```

Might be [related](https://github.com/cpplint/cpplint/pull/235)?

**Comment by aaronliu0130** [comment]
Yes, this is a problem. While we find the time to fix it, you can add `filter=-whitespace/indent_namespace` to a `CPPLINT.cfg` file.
